packages/angular/src/data/content.interface.ts

```diff
@@ -0,0 +1,212 @@
+// I will eventually break this up into individual schemas as we map out the API.
+export interface ContentEntity<t> {
```

Comment:
@apocist I finished the interface so it should pass Travis CI now.

Reply:
Nice~